projects/storefrontapp/src/environments/environment.ts

```diff
@@ -15,7 +15,9 @@ import { Environment } from './models/environment.model';
 
```

Comment:
I do not think that you should commit the change of environment, it has a risk of later forgetting about removing that change

Reply:
We are using this env in epic. So in the end we will change to develop variables.